### ISSUE TITLE
Tensorflow issues with integer inputs

### ISSUE THREAD

**Issue by htjb** [htjb/globalemu#11]
A few users have reported issues with integer redshift inputs not being handled properly by tensorflow because they are not cast to float32s before being passed on. This is an easy fix where all training data/labels and test data/labels are properly cast to float32s in the `preprocess.py` file.
